code/asteroid/asteroid.cpp

```diff
@@ -966,7 +966,7 @@ static void asteroid_aim_at_target(object *objp, object *asteroid_objp, float de
 		vm_vec_add2(&rand_vec, &objp->orient.vec.rvec);
 	vm_vec_normalize(&rand_vec);
 
-	speed = Asteroid_info[0].max_speed * (frand()/2.0f + 0.5f);
+	speed = Asteroid_info[ASTEROID_TYPE_LARGE].max_speed * (frand()/2.0f + 0.5f);
```

Comment:
In the event that non-large asteroids are ever aimed, I suggest this instead:
```suggestion
	auto asp = &Asteroids[asteroid_objp->instance];
	speed = Asteroid_info[asp->asteroid_type].max_speed * (frand()/2.0f + 0.5f);
```

Reply:
good idea - done